full_bubble_model.py

```diff
@@ -468,40 +747,35 @@ def simulate(kwargs):
 
 """________________________________Plotting________________________________"""
 
-# This funfction solves the differential equation, and plots it
-    # cpar: control parameters in a dictionary
-    # t_int: time interval to solve the diffeq in (default: [0, 1] [s])
-    #        graphs will be plotted in this intervall, if not default
-    # n: how long should the plotted time interval be compared to the collapse time (default: 5 [-])
-    # base_name: save plots as .png (default: '' alias do not save)
-    #            use base_name='plot' --> plot_1.png, plot_2.png
-    #            use base_name='images/plot' to save into images folder
-    #            using a folder for images is recommend
-    #            this folder have to be created manually
-    # LSODA_timeout, Radau_timeout: timeout (maximum runtime) for different solvers in solve() in seconds
-def plot(cpar, t_int=np.array([0.0, 1.0]), n=5.0, base_name='', LSODA_timeout=30, Radau_timeout=300):
-    
+def plot(cpar, t_int=np.array([0.0, 1.0]), n=5.0, base_name='', LSODA_timeout=30, Radau_timeout=300, presentation_mode=False, plot_pressure=False, show_legend=False, show_cpar=True):
```

Comment:
Plots are now nicer, and they have more options, like ploting internal pressure and excitation (read describtion).
You might see these multiline comments under function headers. This is the offical way to comment a python function. It is recognized by many IDEs and the interpreter. If you hover over a function with your mouse, this will be shown to you, also acces it as help(de.plot) or print(de.plot.__doc__)